backend/open_webui/routers/images.py

```
@@ -430,47 +454,44 @@ def get_models(request: Request, user=Depends(get_verified_user)):
+                mime_type = r.headers["content-type"]
+                return r.content, mime_type
+            else:
+                log.error("Url does not point to an image.")
+                return None
         else:
-            log.error("Url does not point to an image.")
-            return None
-
+            if "," in data:
+                header, encoded = data.split(",", 1)
+                mime_type = header.split(";")[0].lstrip("data:")
```

Ruff (0.14.7)

Using `.strip()` with multi-character strings is misleading

(B005)